wallet/src/actors/app/methods.rs

```diff
@@ -631,25 +631,37 @@ impl App {
     /// and add a local pending balance movement to the wallet state.
     pub fn send_transaction(
         &self,
-        session_id: &types::SessionId,
-        wallet_id: &str,
+        session_id: types::SessionId,
+        wallet_id: String,
         transaction: types::Transaction,
     ) -> ResponseActFuture<SendTransactionResponse> {
         let f = fut::result(
             self.state
                 .get_wallet_by_session_and_id(&session_id, &wallet_id),
         )
-        .and_then(move |wallet, act: &mut Self, _| {
-            act.send_inventory_transaction(transaction.clone())
-                .and_then(move |jsonrpc_result, _slf, _ctx| {
+        .and_then(move |wallet, slf: &mut Self, _| {
+            slf.send_inventory_transaction(transaction.clone())
+                .and_then(move |jsonrpc_result, slf: &mut Self, _ctx| {
```

Comment:
Let's try to follow the same convention as in the node

```suggestion
                .and_then(move |jsonrpc_result, act, _ctx| {
```